### ISSUE TITLE
Download specific version of object 

### ISSUE THREAD

**Issue by ar-leskoma** [mygrocerydeals/gradle-s3-plugin#14]
S3 buckets have object versioning, so I want to know if I can download a specific version of an object. It's possible with the AWS SDK in Python:
```
s3 = boto3.resource('s3')
bucket = s3.Bucket("mybucket")
bucket.download_file(
    "somefile",
    "/donwload/path/somefile.txt",
    ExtraArgs={"VersionId": "my_version"}
)
```
Where `"my_version"` is the version I want to download. Is that possible with this plugin? If not, is that a feature that can be implemented?

**Comment by peter-thomas-mgd** [comment]
It is not possible to specify a file version to download with the plugin. The plugin will always download the latest version of the file if the bucket is versioned. The underlying plugin uses the AWS Java v2 SDK, so it is definitely possible to add this as a new feature. It would really only make sense as an option for the single file download task (i.e. implement a new optional parameter for the task). Unfortunately, I won't have time to look into this until at least 2 weeks from now. The other alternative is to submit a pull request.

**Comment by peter-thomas-mgd** [comment]
@ar-leskoma This has been implemented in version 2.0.2 of the plugin. See the README.md for details.
